### DISCUSSION TITLE
IP Adapter for Flux

### DISCUSSION THREAD

**Discussion by SAC020** [vladmandic/sdnext#3413] [General]
I've seen in the wiki that it's planned, was wondering a ballpark timeline estimation. Speaking for myself, it's the feature I am using most often and I'd be very happy to have it working for Flux.

Thanks!

**Comment by vladmandic** [comment]
I like ipadapters as well, but no exact time line just yet.

**Reply by SAC020** [reply]
> I like ipadapters as well, but no exact time line just yet.

Hi Vlad,

Just checking if in the meantime you have a tentative timeline for IP adapters (and/or CTRL-X) for Flux.

Thanks!

**Reply by SAC020** [reply]
😄 please

**Reply by SunGreen777** [reply]
His name is not Vlad but Vladimir, Vlad is short for his name.
Looks like CN for flux is a problem, Forge also has a delay

**Reply by vladmandic** [reply]
both flux dev and schnell are distilled models. which is great for ouf-of-the-box aesthetics.
but making any distilled model do exactly what you want instead of what it wants is not easy - its trained to "i know whats best".

**Reply by SAC020** [reply]
> both flux dev and schnell are distilled models. which is great for ouf-of-the-box aesthetics. but making any distilled model do exactly what you want instead of what it wants is not easy - its trained to "i know whats best".

I see. Thank you for your efforts, I'm sure you will figure it out eventually. Flux is great with details, but very frustratingly rigid. Looking forward to any progress.

**Comment by SAC020** [comment]
Hi. Is there any hope for ip adapter for flux?

**Reply by vladmandic** [reply]
just added to dev branch.
